Doc/tutorial/interpreter.rst

```diff
@@ -26,7 +26,7 @@ popular alternative location.)
 On Windows machines, the Python installation is usually placed in
 :file:`C:\\Python36`, though you can change this when you're running the
```

Comment:
This isn't true anymore, is it?

Reply:
No. Currently, the python dir is put, I believe, in Program Files or Program Files (x86) for all users install or somewhere hidden in Appdate for user install.  I am not sure what to substitute.

Reply:
I don't have the Windows experience to suggest a better wording so I'll leave it to someone else to write a PR to address that.